common/src/main/java/io/netty/util/Recycler.java

```diff
@@ -541,6 +554,34 @@ void push(DefaultHandle<?> item) {
             this.size = size + 1;
         }
 
+        private void pushLater(DefaultHandle<?> item, Thread thread) {
+            // we don't want to have a ref to the queue as the value in our weak map
+            // so we null it out; to ensure there are no races with restoring it later
+            // we impose a memory ordering here (no-op on x86)
+            Map<Stack<?>, WeakOrderQueue> delayedRecycled = DELAYED_RECYCLED.get();
+            WeakOrderQueue queue = delayedRecycled.get(this);
+            if (queue == null) {
+                if (delayedRecycled.size() >= maxDelayedQueues) {
+                    delayedRecycled.put(this, WeakOrderQueue.DUMMY); // Add a dummy queue so we know we should drop the
```

Comment:
nit: just move this comment onto the line before so it is on a contiguous line (the inline comment doesn't save any lines if it wraps)